ui/src/actions/types.js

```diff
@@ -1,4 +1,5 @@
 export const REVERSE_SORT = 'REVERSE_SORT';
```

Comment:
will probably rename this also for consistency now that we have 2 reverse sort actions